### PR TITLE
#7240 Extracted one line that underscores the view template file into a method

### PR DESCRIPTION
**Pull request by ghost**
Just extracted one line which inflects the name of view template file into under_scored format to a method, so that who wants to change the rule can override.

### CODE REVIEW

**Comment by markstory** [comment]
This change can't be merged. Could you rebase it on to the current HEAD?

File: src/View/View.php [hunk 1]
```diff
@@ -897,6 +899,18 @@ protected function _getViewFileName($name = null)
     }
 
     /**
+     * Change the name of a view template file into underscored format.
+     *
+     * Extracted so that who wants to change the inflection rule can override.
```
Comment (dereuromark) [inline src/View/View.php]: This sentence is not necessary IMO :)

File: src/View/View.php [hunk 2]
```diff
@@ -897,6 +899,18 @@ protected function _getViewFileName($name = null)
...
+     *
+     * @param string $name Name of file which should be underscored.
```
Comment (dereuromark) [inline src/View/View.php]: "should be inflected" would be more generic.

**Comment by NaranjaWeb** [comment]
I agree those comments by @markstory. You are always encouraging. Thank you.
I could not proceed within the same topic or same pull request, I made another one.
I feel sorry if I added some extra work to you. I dare to close this topic.

**Pull request closed without merging**